less/manta-custom.less

```diff
@@ -539,6 +539,27 @@ ul.nav li.dropdown:hover > .dropdown-onhover {
   }
 }
 
+// Danger and error appear as red
+.btn-brand {
+  .button-variant(@btn-brand-color, @btn-brand-bg, @btn-brand-border);
+}
+
+// Don't change opacity on disabled button
+.opaque {
+  opacity: 1;
+  &.btn.disabled,
+  &.btn[disabled],
+  fieldset[disabled] & {
+    opacity: 1;
+  }
+  &.btn {
+    color: @white;
+  }
+  &.btn-default {
+    color: @gray-dark
```

Comment:
Because a little bit above this, there's a `.btn.disabled` class that sets the text color to some sort of gray, and it looked a little weird with the button color still active but the text itself sort of half grayed out.